qubes/storage/__init__.py

```diff
@@ -921,15 +926,20 @@ def _init(cls, dir_path):
                 if os.getuid():
                     sudo = ['sudo']
                 root_table = subprocess.check_output(sudo + ["dmsetup",
-                    "-j", str(fs_major), "-m", str(fs_minor),
-                    "table"], stderr=subprocess.DEVNULL)
+                                                             "-j",
+                                                             str(fs_major),
+                                                             "-m",
+                                                             str(fs_minor),
+                                                             "table"],
```

Comment:
Can you format it better? Like, break the line after `check_output(`

Reply:
Yes. It should be better.